dd-sdk-android/apiSurface

```diff
@@ -346,5 +347,6 @@ open class com.datadog.android.tracing.TracingInterceptor : okhttp3.Interceptor
   constructor(TracedRequestListener = NoOpTracedRequestListener())
   override fun intercept(okhttp3.Interceptor.Chain): okhttp3.Response
   protected open fun onRequestIntercepted(okhttp3.Request, io.opentracing.Span?, okhttp3.Response?, Throwable?)
+  protected open fun canSendSpan(): Boolean
```

Comment:
Do you want to expose this ? I think it can be a bit weird for someone who's extending the interceptor and can lead to a silent malfunction.